Index: editable.c
===================================================================
--- editable.c	(revision 8954)
+++ editable.c	(working copy)
@@ -267,7 +267,7 @@
 	if (ks == AG_KEY_ESCAPE) {
 		return (0);
 	}
-	if (ks == AG_KEY_RETURN &&
+	if ((ks == AG_KEY_RETURN || ks == AG_KEY_KP_ENTER) &&
 	   (ed->flags & AG_EDITABLE_MULTILINE) == 0)
 		return (0);
 
@@ -900,7 +900,7 @@
 		AG_ScheduleTimeout(ed, &ed->toCursorBlink, agTextBlinkRate);
 		AG_UnlockTimeouts(ed);
 	}
-	if (keysym == AG_KEY_RETURN &&
+	if ((keysym == AG_KEY_RETURN || keysym == AG_KEY_KP_ENTER) &&
 	   (ed->flags & AG_EDITABLE_MULTILINE) == 0) {
 		if (ed->flags & AG_EDITABLE_ABANDON_FOCUS) {
 			AG_WidgetUnfocus(ed);
