*** core.c	2009-12-22 03:53:18.000000000 +0100
--- core.c	2010-10-31 17:54:43.040486700 +0100
***************
*** 33,39 ****
  #include <config/localedir.h>
  #include <config/ag_network.h>
  #include <config/ag_threads.h>
! #include <config/have_gettimeofday.h>
  
  #ifdef AG_THREADS
  #include <config/have_pthreads_xopen.h>
--- 33,39 ----
  #include <config/localedir.h>
  #include <config/ag_network.h>
  #include <config/ag_threads.h>
! //#include <config/have_gettimeofday.h>
  
  #ifdef AG_THREADS
  #include <config/have_pthreads_xopen.h>
***************
*** 101,113 ****
  	AG_RegisterClass(&agDbObjectClass);
  	AG_RegisterClass(&agDbClass);
  
! #if defined(HAVE_GETTIMEOFDAY)
! 	AG_SetTimeOps(&agTimeOps_gettimeofday);
! #elif defined(_WIN32)
! 	AG_SetTimeOps(&agTimeOps_win32);
! #else
  	AG_SetTimeOps(&agTimeOps_dummy);
! #endif
  
  	AG_InitTimeouts();
  	AG_DataSourceInitSubsystem();
--- 101,113 ----
  	AG_RegisterClass(&agDbObjectClass);
  	AG_RegisterClass(&agDbClass);
  
! //#if defined(HAVE_GETTIMEOFDAY)
! 	//AG_SetTimeOps(&agTimeOps_gettimeofday);
! //#elif defined(_WIN32)
! 	//AG_SetTimeOps(&agTimeOps_win32);
! //#else
  	AG_SetTimeOps(&agTimeOps_dummy);
! //#endif
  
  	AG_InitTimeouts();
  	AG_DataSourceInitSubsystem();
